Northstar.CustomServers/mod/scripts/vscripts/burnmeter/_burnmeter.gnut

```diff
@@ -494,44 +495,61 @@ void function PlayerUsesPhaseRewindBurncardThreaded( entity player )
 	for ( int i = positions.len() - 1; i > -1; i-- )
 	{
 		// should looks better?
+		player.UnforceCrouch()
+		player.UnforceStand()

```

Comment:
is this needed?

Reply:
it's needed, for players may forced crouch/stand in their rewind, if we don't clean up it will make them no longer able to stand/crouch

Reply:
This would break though if the player was meant to be force crouch/stand at the beginning (end?) of the rewind data, as they would come out of it unforced. 

I don't think that there is a nice solution for this though, and force crouch/stand is used rarely enough to not matter imo

Reply:
> This would break though if the player was meant to be force crouch/stand at the beginning (end?) of the rewind data, as they would come out of it unforced.
> 
> I don't think that there is a nice solution for this though, and force crouch/stand is used rarely enough to not matter imo

that's true, and I can't find a function like `player.IsForceCrouched()` or something to restore player's force crouch/stand